devify

```diff
@@ -36,8 +36,8 @@ case "$1" in
 *"DualShock"*)
   notify "PS4 Controller" "ps4_controller"
   ;;
-*"Switch Pro"*) # this might be wrong, so please feel free to change it
-  notify "Nintendo Switch Pro Controller" "switch_pro_controller"
+*"Pro Controller"*)
```

Comment:
Nintendo uses the branding "Pro Controller for Nintendo Switch", so "Pro Controller" seemed appropriate for the notification (that's also what the controller reports its name as to my system).